src/loaders/workspace_downloader/workspace_downloader.py

```diff
@@ -0,0 +1,356 @@
+"""
+usage: workspace_downloader.py [-h] --workspace_id WORKSPACE_ID [--root_dir ROOT_DIR] [--kb_base_url KB_BASE_URL] [--workers {1,2,3,4}]
```

Comment:
So it looks like the entire worked choice list shows up here, which for some hardware, will be huge. I don't think that's really workable

Reply:
Addressed above!

Reply:
:+1: